lib/howl/io/process.moon

```diff
@@ -127,7 +148,14 @@ class Process
 
   send_signal: (signal) =>
     signal = signals[signal] if type(signal) == 'string'
-    C.kill(@pid, signal)
+    if jit.os == 'Windows'
```

Comment:
@nilnor Good point. I was thinking that I could add a table `platform` to `howl.sys`. It would have the following methods/attributes/functions/you-get-the-point:

- `platform_tmpname` from `howl/io/file.moon`, under the name `tmpname`.
- `fd_to_stream(win_type, unix_type, fd)` from `howl/io/input_stream.moon` and `howl/io/output_stream.moon`. It could be called like `fd_to_stream(Win32InputStream, UnixInputStream, fd)`, and it would return the correct type, converting the file descriptor as necessary. Alternatively, it could be `fd_to_stream(kind, fd)` and be called like `fd_to_stream('Input', fd)` and automatically index the streams from `ljglibs.io` (like `require 'ljglibs.io'["#{platform}#{kind}Stream"]`), but I'm not sure if you would consider that too magical.
- `default_shell` and `send_signal` from `howl/io/process.moon`.
- `load_font` from the font-loading code in `howl/init.lua`.

I'm not sure whether the font name code in `font_awesome.moon` would really fit in `howl.sys`, though, as well as the file separator handling code in `file.moon`.

How does all that sound?